datacontract/imports/glue_importer.py

```diff
@@ -186,27 +184,43 @@ def create_typed_field(dtype: str) -> Field:
     """
     field = Field()
```

Comment:
Lots of Glue data types are not yet implemented:
- `binary`
- `char`
- `interval`
- `smallint`
- `tinyint`
- `union`
etc.

I am not quite sure how to add them cause the supported list of data types:
```
['number', 'decimal', 'numeric', 'int', 'integer', 'long', 'bigint', 'float', 'double', 'string', 'text', 'varchar', 'boolean', 'timestamp', 'timestamp_tz', 'timestamp_ntz', 'date', 'array', 'map', 'object', 
'record', 'struct', 'bytes', 'null']
```
https://datacontract.com/#data-types

Can I just extend this list of supported data types with the ones above? Thanks!

Reply:
Please try to map Glue Types to a logical data type (https://datacontract.com/#data-types
) that makes most sense. 

You can use a config object to specify the physical type:

```
type: bytes
config:
  glueType: binary
```

Some types (esp. union) may not be supported, add a warning here.

Reply:
Amazing! Thank you, I will keep that in mind for the next PRs